game/script-jokes.rpy

```diff
@@ -631,3 +659,74 @@ label joke_con_crete:
     extend 1nchgn "-crete!"
 
     return
+
+label joke_footless_snakes:
+    n 1ulraj "Hey,{w=0.2} [player]...{w=1}{nw}"
+    extend 1tnmfl " have you ever wondered how to measure snakes?"
```

Comment:
```suggestion
    extend 1tnmfl " have {i}you{/i} ever wondered how snakes are measured?"
```
Just a small tweak - this would work for a single snake (_How to measure a snake?_), but it seemed a little odd for plural snakes.